src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -2800,6 +2814,12 @@ bool QgsPostgresConn::allowRasterOverviewTables( const QString &connName )
   return settings.value( "/PostgreSQL/connections/" + connName + "/allowRasterOverviewTables", true ).toBool();
 }
 
+QString QgsPostgresConn::restrictToSchema( const QString &connName )
```

Comment:
```suggestion
QString QgsPostgresConn::schemaToRestrict( const QString &connName )
```

restrictToSchema name let's think that it actually modify something where it actually only return a schema name

Reply:
Done